src/app/views/teams/team-create/TeamCreate.jsx

```diff
@@ -9,6 +9,7 @@ const propTypes = {
     onCreateSuccess: PropTypes.func.isRequired,
 };
 
+// TODO remove when new AUTH is turned on
 class TeamCreate extends Component {
```

Comment:
does it need to be a class component?

Reply:
This whole 'TeamCreate.jsx' file will be deleted, the new screens replace this entirely, this TODO that has been added is to make clean up later easier to ensure we delete the then unused components